.gitignore

```diff
@@ -5,3 +5,4 @@ storage_data
 logs
 *.log
 .pyenv
+.idea
```

Comment:
This is not needed. It's the developers responsibility to ignore IDE specific files.